app/controllers/users_controller.rb

```diff
@@ -20,11 +26,63 @@ def create
     end
   end
 
+  def edit
+    # @user = User.find_by(id: params[:id])
+    return if @user
+
+    flash[:warning] = t(".error.not_found")
+    redirect_to root_path
+  end
```

Comment:
```suggestion
  def edit; end
```
Chỗ này đã có hàm load_user thực hiện trước khi vào edit r